runtime/development/src/lib.rs

```diff
@@ -1678,6 +1691,44 @@ impl pallet_liquidity_rewards::Config for Runtime {
 	type WeightInfo = ();
 }
 
+frame_support::parameter_types! {
+	pub const BlockRewardsDomain: RewardDomain = RewardDomain::Block;
+	pub const EpocDurationBlockRewards: BlockNumber = 2 * Period::get();
+
+	// TODO: Convert to common consts
+	#[derive(scale_info::TypeInfo, Debug, PartialEq, Clone)]
+	// TODO: Ensure uniqueness, currently abbreviates "blockrewards/collator"
+	pub const CollatorCurrencyId: u32 = u32::from_be_bytes(*b"br/c");
+	#[derive(scale_info::TypeInfo, Debug, PartialEq, Clone)]
+	pub const CollatorGroupId: u32 = 1u32;
+	#[derive(scale_info::TypeInfo, Debug, PartialEq, Clone)]
+	pub const DefaultCollatorStake: Balance = 1000 * CFG;
+
+	pub const RemainingRewardCollector: Option<PalletId> = Some(TreasuryPalletId::get());
+}
+
+impl pallet_block_rewards::Config for Runtime {
+	type AdminOrigin = EnsureRootOr<HalfOfCouncil>;
+	type AuthorityId = AuraId;
+	type Balance = Balance;
+	type CollatorCurrencyId = CollatorCurrencyId;
+	type CollatorGroupId = CollatorGroupId;
```

Comment:
I think there is no need to specify this from the outside. It can be a constant inside the pallet (?).